examples/chat/tui/commands.nim

```diff
@@ -175,6 +263,23 @@ proc command*(self: ChatTUI, command: Disconnect) {.async, gcsafe, nimcall.} =
   else:
     self.wprintFormatError(getTime().toUnix(), "client is not online.")
 
+# GetCustomTokens -----------------------------------------------------------------
+
+proc help*(T: type GetCustomTokens): HelpText =
+  let command = "getcustomtokens"
+  HelpText(command: command, aliases: aliased[command], description: "Lists " &
+    "all existing custom tokens.")
+
+proc new*(T: type GetCustomTokens, args: varargs[string]): T =
+  T()
+
+proc split*(T: type GetCustomTokens, argsRaw: string): seq[string] =
+  @[]
+
+proc command*(self: ChatTUI, command: GetCustomTokens) {.async, gcsafe, nimcall.} =
+  asyncSpawn self.client.getCustomTokens()
```

Comment:
Do we need a try/except surrounding the `getCustomTokens` call?

Reply:
Looked at the body of StatusObject.getCustomTokens, and it seems to have try...except inside. It shouln't throw exceptions it seems. Should i modify it so that it does?

Reply:
Actually that's a good point. We won't be able to get reliable exceptions from calls that are `asyncSpawn`ed, however I have seen it work, but while it may work sometimes, apparently not all exception are guaranteed to be catchable here. We may want to remove the other `try..except`s for `command.DeleteCustomToken` and `command.AddCustomToken` for consistency's sake then.

Reply:
Done.